index.js

```diff
@@ -32,10 +54,12 @@ export default class DashShakaPlayback extends HTML5Video {
   }
 
   play() {
-    !this._player && this._setup()
+    if (!this._player) {
```

Comment:
yeah good point.

i guess it was just a quick walk through of things i though were ugly and wouldn't pass my linter rules.